Extensions/AnchorBehavior/anchorruntimebehavior.ts

```diff
@@ -89,14 +89,25 @@ namespace gdjs {
         gdjs.AnchorRuntimeBehavior.prototype.doStepPreEvents
       ) as FloatPoint;
       // TODO EBO Make it work with event based objects or hide this behavior for them.
-      const game = instanceContainer.getGame();
-      let rendererWidth = game.getGameResolutionWidth();
-      let rendererHeight = game.getGameResolutionHeight();
+      let parentMinX = instanceContainer.getUnrotatedViewportMinX();
+      let parentMinY = instanceContainer.getUnrotatedViewportMinY();
+      let parentMaxX = instanceContainer.getUnrotatedViewportMaxX();
+      let parentMaxY = instanceContainer.getUnrotatedViewportMaxY();
```

Comment:
This looks risky but `_cachedGameResolutionWidth` is what is actually used by `layer.convertCoords`.